### DISCUSSION TITLE
Why `.md` files does not have Copyright Licenses?

### DISCUSSION THREAD

**Discussion by pritidesai** [tektoncd/pipeline#4626] [Q&A]
@jerop brought up that we do not have copyright licenses in any of the files under [docs](https://github.com/tektoncd/pipeline/tree/main/docs) folder:

https://github.com/tektoncd/pipeline/pull/4600/files#r815156707

I found this weird and wondering why the licenses are missing 🤔 

We have added such licenses to the Apache project I have contributed before and they were added since the foundation mandated them. I am not sure what are the guidelines here.

**Comment by vdemeester** [comment]
Note: I think that by default the main [LICENSE][./LICENSE] applies when there is no license header in files.
